src/generic/model-store/slice.js

```diff
@@ -75,3 +75,5 @@ export const {
 } = slice.actions;
 
 export const { reducer } = slice;
+
+window.act = slice.actions;
```

Comment:
This is just to dispatch events using the browser console so I can test things during development. It will be removed before merge.